MAUI/SmartDataGrid/Appearance.md

```diff
@@ -0,0 +1,512 @@
+
+{% tabs %}
+{% highlight xaml %}
+<smart:SfSmartDataGrid ItemsSource="{Binding OrderInfoCollection}">
+    <smart:SfSmartDataGrid.ToolbarTemplate>
+      <DataTemplate>
+          <Grid BackgroundColor="LightGray" Padding="8,8" ColumnDefinitions="*,Auto">
+              <Label Text="My Custom Toolbar" 
+                     VerticalTextAlignment="Center"
+                     FontAttributes="Bold" >
+              </Label>
+              <Button Text="Ask AI" 
+                      Grid.Column="1" 
+                      Clicked="OnAskAIClicked" >
+              </Button>
+          </Grid>
+      </DataTemplate>
+    </smart:SfSmartDataGrid.ToolbarTemplate>
+</smart:SfSmartDataGrid>
+{% endhighlight %}
+
+{% highlight c# %}
+SmartGrid.ToolbarTemplate = new DataTemplate(() =>
+{
+    var grid = new Grid
+    {
+        BackgroundColor = Colors.LightGray,
+        Padding = new Thickness(8, 8),
+        ColumnDefinitions =
+        {
+            new ColumnDefinition { Width = GridLength.Star },
+            new ColumnDefinition { Width = GridLength.Auto }
+        }
+    };
+
+    var label = new Label
+    {
+        Text = "My Custom Toolbar",
+        VerticalTextAlignment = TextAlignment.Center,
+        FontAttributes = FontAttributes.Bold
+    };
+
+    var button = new Button
+    {
+        Text = "Ask AI"
+    };
+
+    Grid.SetColumn(button, 1);
+
+    button.Clicked += OnAskAIClicked;
+
+    grid.Add(label, 0, 0);
+    grid.Add(button, 1, 0);
+
+    return grid;
+});
+{% endhighlight %}
+{% endtabs %}
+
+
+## AssistView Button
+
+### Styling
+
+The AssistView button’s visual style is driven by the following `SmartAssistStyle` properties:
+
+- `AssistButtonBackground`: Background color for the AssistView button.
+- `AssistButtonIconColor`: Color applied to the AssistView button icon.
+- `AssistButtonCornerRadius`: Corner radius for the button shape.
+
+{% tabs %}
+{% highlight xaml %}
+<smart:SfSmartDataGrid ItemsSource="{Binding OrderInfoCollection}">
+  <smart:SfSmartDataGrid.AssistViewSettings>
+    <smart:DataGridAssistViewSettings>
+      <smart:DataGridAssistViewSettings.AssistStyle>
+        <smart:SmartAssistStyle
+          AssistButtonBackground="#6750A4"
+          AssistButtonIconColor="#FFFFFF"
+          AssistButtonCornerRadius="10" />
+      </smart:SmartAssistStyle>
+    </smart:DataGridAssistViewSettings.AssistStyle>
+    </smart:DataGridAssistViewSettings>
+  </smart:SfSmartDataGrid.AssistViewSettings>
+</smart:SfSmartDataGrid>
+{% endhighlight %}
+
+{% highlight c# %}
+var style = SmartGrid.AssistViewSettings.AssistStyle;
+style.AssistButtonBackground   = Color.FromArgb("#6750A4");
+style.AssistButtonIconColor    = Colors.Red
+style.AssistButtonCornerRadius = 10;
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="maui-smart-datagrid-assist-button-style" src="Images\Appearance\maui-smart-datagrid-assist-button-style.png" width="600"/>
+
```

Comment:
Can you include a note section here to provide details to customize the AIAssistView control using their help document? It helps users to modify the chat appearances inside the AssistView control.

https://help.syncfusion.com/maui/aiassistview/styles